dataplane/src/main.rs

```diff
@@ -57,6 +59,12 @@ fn main() {
     /* parse cmd line args */
     let args = CmdArgs::parse();
 
+    let grpc_address = "[::1]:50051".parse().expect("Bad grpc address");
```

Comment:
Should the address and port be configurable? (I guess this is for a follow-up?)

Reply:
@sergeymatov is going to look into this. We did not bother changing it since we may use unix sockets soon.